src/Symfony/Contracts/Deprecation/deprecated.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Triggers a deprecation.
+ *
+ * As recommended for prod, turn the "zend.assertions" ini setting to 0 or -1 to disable deprecation notices.
+ *
+ * The function doesn't use type hints nor variadics on purpose, to make it as fast as possibly achievable.
```

Comment:
```suggestion
 * The function doesn't use type declarations nor variadics on purpose, to make it as fast as possibly achievable.
```

Reply:
> The function doesn't use type hints nor variadics on purpose

The statement seems wrong. It does use variadics `...$args`

Reply:
fixed!